lib/internal/http2/core.js

```diff
@@ -3399,6 +3397,18 @@ binding.setCallbackFunctions(
   onStreamClose
 );
 
+const classDefinitionsPrivate = {
+  ClientHttp2Stream,
+  ServerHttp2Stream
+};
+const classDefinitionsPublic = {
+  Http2Session,
+  Http2Stream,
+  ServerHttp2Session,
+  Http2ServerRequest,
+  Http2ServerResponse
+};
```

Comment:
same here - anything defined after it's referenced violates the intent of the rule.